src/metabase/models/metric.clj

```diff
@@ -65,7 +65,7 @@
 
 ;; ## Persistence Functions
 
-(defn create-metric
+(defn create-metric!
```

Comment:
I've seen it referenced in a variety of contexts, but I was going off of https://github.com/bbatsov/clojure-style-guide which says 

> The names of functions/macros that are not safe in STM transactions should end with an exclamation mark (e.g. reset!).

In any case, even if the purpose is side effects I'm not convinced that functions which change the Metabase application state really apply.  We have a lot of functions which are for updating something about the application database and I don't think we really need to suffix all of those with exclamation marks.